oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/zabbix.yml

```diff
@@ -0,0 +1,157 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# UI templates initialized file includes the default template when the SkyWalking OAP starts up at the first time.
+#
+# Also, SkyWalking would detect the existing templates in the database, once they are missing, all templates in this file
+# could be added automatically.
+
+templates:
+  - name: "Zabbix"
```

Comment:
Do you really need this? My expectation would be, the VM monitoring could share the same templates with existing VM template. Could you share the details about why adding this?

Reply:
There has a little difference between OTEL or Zabbix. Such as `Disk R/W`, the Zabbix is not supporting read bytes per second, only support read/write count. I found the [Zabbix agent support platform document](https://www.zabbix.com/documentation/current/manual/appendix/items/supported_by_platform), It could behave more description.

Reply:
![image](https://user-images.githubusercontent.com/5441976/107146405-f7439b00-6982-11eb-9076-f175001548dd.png)

I can see the supports of read and write, what do I miss?

Reply:
That's true, but the `type` parameter value `bytes` Linux is not supported.
![image](https://user-images.githubusercontent.com/3417650/107150509-0af9fc00-6999-11eb-9e98-4918913e2d66.png)
![image](https://user-images.githubusercontent.com/3417650/107150519-1f3df900-6999-11eb-9e1f-01835c4e818c.png)

Reply:
Let's merge sps into the existing the disk metrics. We could declare them clearly on the UI configuration, rather than providing a totally new dashboard.